pkg/service/group.go

```diff
@@ -204,6 +207,14 @@ func (s *groupService) createGroupRoleRelations(ctx context.Context, db bun.IDB,
 			}
 
 			namespace := pnr.GetNamespace()
+			match := regexc.MatchString(namespace)
+			if match {
+				return &userv3.Group{}, nil, fmt.Errorf("namespace %q is invalid", namespace)
+			}
+			if !(len(namespace) >= 1 && len(namespace) <= 63) {
```

Comment:
can we simplify the condition check here

len(namespace) < 1 || len(namespace) > 63

Reply:
Corrected